apps/backend/Dockerfile

```
@@ -0,0 +1,66 @@
+cat > "$CLAUDE_CONFIG_DIR/settings.json" <<EOF\n\
+{\n\
+  "apiBaseUrl": "${ANTHROPIC_BASE_URL:-https://api.anthropic.com}",\n\
+  "apiKey": "$API_KEY",\n\
+  "models": {\n\
+    "haiku": "${ANTHROPIC_DEFAULT_HAIKU_MODEL:-${DEFAULT_HAIKU_MODEL:-claude-haiku-4-5-20251001}}",\n\
+    "sonnet": "${ANTHROPIC_DEFAULT_SONNET_MODEL:-${DEFAULT_SONNET_MODEL:-claude-sonnet-4-5-20250929}}",\n\
+    "opus": "${ANTHROPIC_DEFAULT_OPUS_MODEL:-${DEFAULT_OPUS_MODEL:-claude-opus-4-5-20251101}}"\n\
+  }\n\
+}\n\
+EOF\n\
+\n\
```

Hadolint (2.14.0)

[error] unexpected 'n'
expecting a new line followed by the next instruction

(DL1000)